projects/landing-page/src/App.tsx

```diff
@@ -145,17 +145,13 @@ const App: React.FunctionComponent = () => {
               title='Next Project'
               imageProps={{path:YourProject}}
             >
-              <SectionRef href='https://github.com/paritytech/substrate/blob/master/docs/CONTRIBUTING.adoc#rules'>
+              <SectionRef href='https://github.com/paritytech/substrate-connect/blob/master/CONTRIBUTING.md'>
                 Contributor’s guide
               </SectionRef>
             </CardProject>
           </Section>
           <ThemeProvider theme={createMuiTheme(dark)}>
             <Section pt={5} pb={5}>
-              <SectionHeading id='playground' prefix='6'>Playground</SectionHeading>
-              <SectionText>Save logged API as global variable. Call methods</SectionText>
-              <SectionHeroText>cmd + alt + i</SectionHeroText>
-              <SectionHeroText>ctrl + shift + i</SectionHeroText>
               {/* TODO: Playground */}
               <Box pt={25}>
```

Comment:
just remove `pt={25}` here. it stands for `padding-top: 8px * 25`

Reply:
That's fixed.  Looks much better